.github/workflows/pull-request.yml

```
@@ -18,6 +15,25 @@ on:
 
 env:
   CODECOV_UNIQUE_NAME: CODECOV_UNIQUE_NAME-${{ github.run_id }}-${{ github.run_number }}
+  DATABASE_NAME: switchmap_unittest
+  DATABASE_USERNAME: switchmap_unittest
+  DATABASE_PASSWORD: switchmap_unittest
+
+services:
```

actionlint (1.7.4)

unexpected key "services" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)